files/default/dism_features.rb

```diff
@@ -0,0 +1,45 @@
+#
+# Author:: Wade Peacock <wade.peacock@visioncritical.com>
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Ohai.plugin(:DismFeatures) do
```

Comment:
Ditto: i plan on submitting the plugin to the ohai project, so it can be removed in the future.